src/ts/components/subtitlesettings/fontfamillyselectbox.ts

```diff
@@ -0,0 +1,81 @@
+import {SelectBox} from '../selectbox';
+import {ListSelectorConfig} from '../listselector';
+import {UIInstanceManager} from '../../uimanager';
+import {SubtitleOverlay} from '../subtitleoverlay';
+import {StorageUtils} from '../../utils';
+
+/**
+ * A select box providing a selection of different font familly.
+ */
+export class FontFamillySelectBox extends SelectBox {
+
+  private overlay: SubtitleOverlay;
+
+  private fontFamilly: { [key: string]: string} = {
+    'default': 'default',
+    'monospaced serif': '"Courier New",Courier,"Nimbus Mono L","Cutive Mono",monospace',
+    'proportional serif': '"Times New Roman",Times,Georgia,Cambria,"PT Serif Caption",serif',
+    'monospaced sans serif': '"Deja Vu Sans Mono","Lucida Console",Monaco,Consolas,"PT Mono",monospace',
+    'proportional sans serif': 'Roboto,"Arial Unicode Ms",Arial,Helvetica,Verdana,"PT Sans Caption",sans-serif',
+    'casual': '"Comic Sans MS",Impact,Handlee,fantasy',
+    'cursive': '"Monotype Corsiva","URW Chancery L","Apple Chancery","Dancing Script",cursive',
+    'small capital': 'unset',
+  }
+
+  private fontVariant: { [key: string]: string} = {
+    'default': 'unset',
+    'monospaced serif': 'unset',
+    'proportional serif': 'unset',
+    'monospaced sans serif': 'unset',
+    'proportional sans serif': 'unset',
+    'casual': 'unset',
+    'cursive': 'unset',
+    'small capital': 'small-caps',
+  }
+
+  private fontStyle: { [key: string]: string} = {
+    'default': 'unset',
+    'monospaced serif': 'unset',
+    'proportional serif': 'unset',
+    'monospaced sans serif': 'unset',
+    'proportional sans serif': 'unset',
+    'casual': 'unset',
+    'cursive': 'italic',
+    'small capital': '',
+  }
+
+  constructor(config: ListSelectorConfig = {}, overlay: SubtitleOverlay) {
+    super(config);
+    this.overlay = overlay;
+  }
+
+  configure(player: bitmovin.PlayerAPI, uimanager: UIInstanceManager): void {
+    super.configure(player, uimanager);
+
+    this.addItem('default', 'default');
```

Comment:
It probably makes sense to use the array index as `key` to make lookup for `FontFamily` objects in `onItemSelected` more straightforward.